lib/signs_ui/config/expiration.ex

```diff
@@ -72,7 +92,19 @@ defmodule SignsUi.Config.Expiration do
     end
   end
 
-  defp expire_single_sign(_, _) do
+  defp expire_single_sign(
+         {id, %Sign{config: %{alert_id: alert_id}} = sign},
+         _current_dt,
+         alert_ids
+       ) do
+    if is_nil(alert_id) or MapSet.member?(alert_ids, alert_id) do
```

Comment:
Can you please move the "not nil" check into the function header to be a guard, like in the `:expires` clause?